simulator/network-reduction/src/main/java/com/powsybl/incubator/simulator/networkreduction/ReductionEngine.java

```diff
@@ -472,7 +472,7 @@ private void defineZones(LfNetwork network) {
     }
 
     // Example to compute full sytem nodal current injectors I = Y*V
-    public void computeCurrentInjections(AdmittanceEquationSystem.AdmittanceType admittanceType) {
+    /*public void computeCurrentInjections(AdmittanceEquationSystem.AdmittanceType admittanceType) {
```

Comment:
To fix before merging this PR?

Reply:
I commented it because it is not used and decreases the code coverage, I can try to put it in a test in a next commit